forem/forem
# [Small fix] Makes stats page responsive

## What type of PR is this? (check all applicable)


## Description

Due to how visible stats are (there's a link directly in Posts that the authenticated user has authored) I put together a small fix to make it look okay in small screens.

This is not meant to be a perfect solution, because that will require time and effort. The objective is to cleanup the screen a bit so it's usable in smaller screens

## Related Tickets & Documents

#13156

## QA Instructions, Screenshots, Recordings

1. Boot app locally and login as `admin@forem.local` (any user with published articles would work)
2. Go into an Article you've authored and tap on "Stats"
3. Make sure this page looks okay in a laptop/desktop browser and also in a smaller screen (DevTools would likely be enough)
4. Make sure the authenticated user is part of an Organization and go to your `/dashboard`
5. Check out the Analytics for the Organization works as well just like you did in Step 3

#### Comparison of existing Stats dashboard in DEV vs this PR:

https://user-images.githubusercontent.com/6045239/114261386-824b0c80-9997-11eb-8dc2-0209f700a663.mov


### UI accessibility concerns?

I believe there could be accessibility issues in this page but since this isn't a prioritized task I want to avoid dedicating too much time into it.

## Added tests?

- [x] No, and this is why: This is most likely a temporary fix for usability purposes. The stats page will most likely get more time/effort assigned in the future (including design).

## [Forem core team only] How will this change be communicated?

- [x] This change does not need to be communicated, and this is why not: Small bugfix/enhancement that doesn't affect any creator/support/dev workflow

## [optional] Are there any post deployment tasks we need to perform?

None

## [optional] What gif best describes this PR or how it makes you feel?

![tetris](https://media.giphy.com/media/5OjI2vCzxo2kM/giphy.gif)
